rosbridge_library/src/rosbridge_library/capabilities/advertise_service.py

```diff
@@ -48,6 +48,13 @@ async def handle_request(self, req, res):
         try:
             return await future
         finally:
+            self.protocol.send(
+                {
+                    "op": "service_response",
+                    "id": request_id,
+                    "result": False,
+                }
+            )
```

Comment:
wasn't sure if this was needed, but I needed it to reproduce the old tests' behavior -- but maybe the tests are outdated